Task 3/moneyexchange/moneychange.md

```diff
@@ -0,0 +1,8 @@
+problem statement - You have coins of 5 and 10 rupees and a particular amount has to be paid. find the minimum number of coins required to pay the amount.
```

Comment:
Have a look at the `README.md` again. Task 3 does not require you to have a `question.md` file. Maybe you should move this to the `Task 1 folder`.